app/src/main/java/com/pvp/app/api/Configuration.kt

```diff
@@ -3,5 +3,4 @@ package com.pvp.app.api
 interface Configuration {
 
     val googleOAuthClientId: String
-    val channelNotificationTasksReminderId: String
```

Comment:
Why? This is a configuration, hard-coded into code for now, but still, a configuration to define a name of task notifications.

To fix the NotificationReceiver error you just had to put `@Inject` annotation next to specified primary constructor `class NotificationReceiver @Inject constructor(...)`, since Dagger didn't know where (and how, because of it) to inject.